packages/gatsby-theme-newrelic/README.md

```diff
@@ -1329,6 +1368,41 @@ const MyComponent = () => {
 };
 ```
 
+### `usePrevious`
+
+A hook that gets the previous state of a stateful value. Useful inorder to do comparing if the state has changed.
+
+```js
+import { usePrevious } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Arguments**
+
+- `value` _(any)_: Value to track its' previous state.
```

Comment:
```suggestion
- `value` _(any)_: Value to track it's previous state.
```

Reply:
Actually I think it's neither of these, I think it is `its` 😆

Reply:
Ha you're right! Apparently my brain was not working when I suggested this haha.